sdk/state/state.go

```diff
@@ -127,8 +127,8 @@ const (
 	StateError State = iota - 1
 	StateNone
 	StateOpen
```

Comment:
when fixing the test I realized these actually were out of order, according to how the state is returned by the sequence number value, `StateClosingWithOutdatedState` has an earlier seqNum than `StateClosing`

Reply:
Ah nice, makes sense 👍🏻.